Sources/StreamChatUI/ChatChannelList/ChatChannelSwipeableListItemView_Tests.swift

```diff
@@ -0,0 +1,118 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelSwipeableListItemView_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        let view = ChatChannelSwipeableListItemView().withoutAutoresizingMaskConstraints
+        view.addSizeConstraints()
+        
+        // Simulate view moving to superview to have view initialized.
+        view.embedInSuperview()
+        
+        // Buttons are revealed after swipe gesture so we need to simulate it to check the buttons.
+        view.trailingConstraint?.constant = 50
+        
+        AssertSnapshot(view, variants: .onlyUserInterfaceStyles)
+    }
+    
+    func test_appearanceCustomization_usingUIConfig() {
+        var config = UIConfig()
+        config.colorPalette.alert = .green
+        
+        let view = ChatChannelSwipeableListItemView().withoutAutoresizingMaskConstraints
+        
+        view.uiConfig = config
+        view.addSizeConstraints()
+
+        // Simulate view moving to superview to have view initialized.
+        view.embedInSuperview()
+        
+        // Buttons are revealed after swipe gesture so we need to simulate it to check the buttons.
+        view.trailingConstraint?.constant = 50
```

Comment:
Can we create some test-only extension for this like `simulateOpen()` or `swipeOpen()`?